server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -59,17 +59,22 @@ const orderMutations = {
         throw new Error('Attaching multiple taxes is not supported yet');
       }
 
-      const getOrderTotalAmount = ({ platformContributionAmount, taxes, quantity }) => {
+      const getOrderTotalAmount = ({ platformTipAmount, platformContributionAmount, taxes, quantity }) => {
         let totalAmount = getValueInCentsFromAmountInput(order.amount) * quantity;
-        totalAmount += platformContributionAmount ? getValueInCentsFromAmountInput(platformContributionAmount) : 0;
+        totalAmount +=
+          platformTipAmount || platformContributionAmount
+            ? getValueInCentsFromAmountInput(platformTipAmount || platformContributionAmount)
+            : 0;
         totalAmount += taxes?.[0].amount ? getValueInCentsFromAmountInput(taxes[0].amount) : 0;
         return totalAmount;
       };
 
       const { order } = args;
-      const { platformContributionAmount } = order;
+      const { platformTipAmount, platformContributionAmount } = order;
       const tax = order.taxes?.[0];
-      const platformFee = platformContributionAmount && getValueInCentsFromAmountInput(platformContributionAmount);
+      const platformFee =
```

Comment:
```suggestion
      const platformTip =
```

Reply:
Thanks. Renamed this one. 👍🏽

Reply:
Sorry, I realized renaming this is a bit harder than thought, since it effects the functions down the stream that uses this value. The point of this PR is to deprecate `platformContributionAmount`. The deprecation and renaming of `platformFee` has a separate issue, https://github.com/opencollective/opencollective/issues/5049 so I suggest we handle this there not to make this one too complicated. 🤔